packages/design-system/src/components/Form/Primitives/_sharedStyles.scss

```diff
@@ -0,0 +1,93 @@
+@use '~@talend/design-tokens/lib/tokens';
```

Comment:
I would rename to _mixins.scss because it doesn't share any styles

Reply:
It doesn't share any mixins either, just tokens